build/cqs.tsv

```diff
@@ -1,19 +1,25 @@
+asm:asm	3.1	compile
 asm:asm	3.3.1	compile
 com.101tec:zkclient	0.3	compile
+com.101tec:zkclient	0.7	compile
+com.101tec:zkclient	0.8	compile
 com.adobe.xmp:xmpcore	5.1.2	compile
 com.beust:jcommander	1.48	compile
 com.clearspring.analytics:stream	2.9.2	compile
 com.drewnoakes:metadata-extractor	2.8.1	compile
 com.esotericsoftware.kryo:kryo	2.21	compile
 com.esotericsoftware.minlog:minlog	1.2	compile
 com.esotericsoftware.reflectasm:reflectasm	shaded:1.07	compile
+com.github.stephenc.findbugs:findbugs-annotations	1.3.9-1	compile
 com.googlecode.efficient-java-matrix-library:core	0.26	compile
 com.google.code.gson:gson	2.3.1	compile
 com.googlecode.json-simple:json-simple	1.1	compile
 com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
+com.google.guava:guava	12.0.1	compile
 com.google.guava:guava	14.0.1	compile
 com.google.guava:guava	17.0	compile
 com.google.guava:guava	18.0	compile
+com.google.protobuf:protobuf-java	2.5.0	compile
```

Comment:
We don't have a completed CQ for protobuf-java.

Also, our CQ is for 2.6.1 (likely from @elahrvivaz's OSM PR).